package.json

```diff
@@ -55,6 +55,7 @@
     "@semantic-release/github": "8.0.7",
     "@semantic-release/npm": "9.0.2",
     "@semantic-release/release-notes-generator": "10.0.3",
+    "@types/facebook-js-sdk": "^3.3.9",
```

Comment:
What's the purpose of this dep?

Reply:
There's the file `FacebookUtils.ts` where the `FB` global is used.